pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/flags.go

```diff
@@ -148,6 +148,8 @@ func ConfigToFlags(kubeAPIServerConfig *kubecontrolplanev1.KubeAPIServerConfig)
 	configflags.SetIfUnset(args, "tls-sni-cert-key", sniCertKeys(kubeAPIServerConfig.ServingInfo.NamedCertificates)...)
 	configflags.SetIfUnset(args, "secure-port", portString)
 
+	configflags.SetIfUnset(args, "feature-gates", "CustomResourcePublishOpenAPI=true")
```

Comment:
this is not good. It means that `CustomResourcePublishOpenAPI` is disabled as soon as another actor (user, operator) enables/disables feature gates.

Reply:
Those flags must be merged, probably just by appending the strings (first this default, then the user provides gates; the later overriding the former).